steps/apply.ts

```diff
@@ -24,14 +26,10 @@ export const enterCrn = async (page: Page, crn: string) => {
   await crnPage.clickSave()
 }
 
-export const completeFundingInformationTask = async (page: Page, name: string) => {
-  const taskListPage = new TaskListPage(page)
-  await taskListPage.clickTask('Add funding information')
+export const completeAreaAndFundingSection = async (page: Page, name: string) => {
+  await completeFundingInformationTask(page, name)
+}
 
-  const fundingInformationPage = await ApplyPage.initialize(
-    page,
-    `How will ${name} pay for their accommodation and service charge?`,
-  )
-  await fundingInformationPage.checkRadio('Personal money or savings')
-  await fundingInformationPage.clickSave()
+export const completeAboutThePersonSection = async (page: Page, name: string) => {
+  await completeWillAnswerEqualityQuestionsTask(page, name)
```

Comment:
sorry I don't have time for a full review, but I was just alerted by Harriet's comment via email. This should be completeEqualityAndDiversityMontitoringTask, shouldn't it? That's the task, and willAnswerEqualityQuestions is a page within that task, I think